.travis.yml

```diff
@@ -23,7 +27,19 @@ before_script:
   - git clone git://github.com/silverstripe-labs/silverstripe-travis-support.git ~/travis-support
   - php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss
   - cd ~/builds/ss
+  - composer install
 
 script:
   - ($HOME/bin/tika-rest-server &) &> /dev/null
   - vendor/bin/phpunit --verbose textextraction/tests/
+  - wget https://scrutinizer-ci.com/ocular.phar
+  - git remote rm origin
+  - git remote add origin git@github.com:silverstripe-labs/silverstripe-textextraction.git
```

Comment:
Should probably remove cover coverage reporting stuff (so this remote and ocular stuff).